### DISCUSSION TITLE
Nested selects

### DISCUSSION THREAD

**Discussion by mfein2** [datapane/datapane#147] [Help] [answered]
I'm using nested selects to have tabs within a tab, however, I'm unable to affix a label to the top level tabs (they are default to section 1 and section 2).  Is it possible to rename these tabs?

**Comment by mands** [comment, marked as answer]
Could you provide a code example? Essentially the `label` attribute needs to be added to the nested blocks within the select, e.g. something like the following should work,

```python
blocks = [
    dp.Select(
        dp.Select(
            dp.Text("...", label="nested select"), 
            dp.Text("...", label="nested select 2"),
            label="top select"),
        dp.Text("...", label="top select 2")
    )
]

dp.Report(blocks=blocks).save("test.html")
```
